ansible/scripts/deploy-local-cluster.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
```

Comment:
Since when? Running the ./hack/verify-all.sh with "All rights reserved." ends with en error.